hp/elitebook/2560p/default.nix

```diff
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  xcfg = config.services.xserver;
+in
+{
+  imports = [
+    ../../../common/cpu/intel
```

Comment:
By including this profile, you are currently enabling intel graphics unconditionally.

Reply:
See above comment.